bosh-upload-release/task.yml

```diff
@@ -0,0 +1,28 @@
+---
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: relintdockerhubpushbot/cf-deployment-concourse-tasks
+    tag: v7.5.0
+
+inputs:
+- name: cf-deployment-concourse-tasks  # - This repo
+- name: bbl-state  # - The repo containing the Director's bbl state dir
+- name: tarballs  # - Directory containing the tarball to upload to the director
```

Comment:
This input name seems confusing. Since the task only works with a single release, there doesn't seem to be a reason for it to be pluralised. How about `release-tarball`?

Reply:
In our case we're passing in an S3 resource that has a lot of tarballs. But `release-tarball` is meaningful in either case.

Reply:
Thanks for making this change.